torch/onnx/symbolic_opset9.py

```diff
@@ -835,24 +835,49 @@ def _prepare_onnx_paddings(dim, pad):
     paddings = paddings[-2::-2] + paddings[-1::-2]
     return paddings
 
+def _convert_padding_node(padding):
+    if sym_help._is_value(padding):
+        if padding.node().kind() == 'onnx::Constant':
+            tval = padding.node()['value']
+            padding = [int(v) for v in tval]
+        elif padding.node().kind() == 'prim::ListConstruct':
```

Comment:
can you simplify the code by using _unpack_list: https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_helper.py#L109
to get the list values from prim::ListConstruct?